compute/kubernetes/api-cli/creating-managing-kubernetes-lifecycle-cliv2.mdx

```diff
@@ -171,7 +171,7 @@ Kubeconfig for cluster your-cluster-ID successfully written at /Users/youruserna
 
 ## Adding a pool to a K8s cluster 
 
-A pool is a set of identical Nodes. A pool has a name, a size (its current number of nodes), nodes number limits (min and max), and a Scaleway Instance type (minimum DEV1-M). Changing these limits increases/decreases the size of a pool. Thus, when autoscaling is enabled, the pool will grow or shrink inside those limits, depending on its load.
+A pool is a set of identical Nodes. A pool has a name, a size (its current number of nodes), nodes number limits (min and max), and a Scaleway Instance type, note that DEV1-S, PLAY2-PICO and STARDUST specs (<2GiB RAM) are too small to be used in a pool). Changing these limits increases/decreases the size of a pool. Thus, when autoscaling is enabled, the pool will grow or shrink inside those limits, depending on its load.
```

Comment:
```suggestion
A pool is a set of identical Nodes. A pool has a name, a size (its current number of nodes), nodes number limits (min and max), and a Scaleway Instance type. 

 <Message type="note">

DEV1-S, PLAY2-PICO and STARDUST specs (<2GiB RAM) are too small to be used in a pool). Changing these limits increases/decreases the size of a pool. Thus, when autoscaling is enabled, the pool will grow or shrink inside those limits, depending on its load).

</Message>
```

Reply:
@nedaehm any thoughts ? I changed the text to be coherent with Developer Docs.

Reply:
For more clarity, I would specify what the Instances are not eligible for in the note, but lgtm otherwise! 👍🏼 @louisportay